cli/tests/snapshot/snapshots/snapshot__eval_stderr_record_access_suggestion.ncl.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: cli/tests/snapshot/main.rs
+expression: err
+---
+error: missing field `some_fild_nam`
+  ┌─ [INPUTS_PATH]/errors/record_access_suggestion.ncl:3:1
+  │    
+3 │ ╭ ╭ ({
+4 │ │ │   some_field_name = "some value",
+5 │ │ │   another_field_name = "another value",
+6 │ │ │ }).some_fild_nam
+  │ ╰─│────────────────^ this requires the field `some_fild_nam` to exist
+  │   ╰──' field `some_fild_nam` is missing here
```

Comment:
I know this was here previously, but to me "field `some_fild_nam` is missing here" suggests that the field is supposed to be present but isn't. I think something like "this record lacks the field `some_fild_nam`" might be clearer in a situation like this.